service/configure.ac

```diff
@@ -195,6 +210,25 @@ if test "x$with_levelzero" != x; then
   EXTRA_LDFLAGS="$EXTRA_LDFLAGS -L$with_levelzero/lib64"
 fi
 
+AC_ARG_WITH([liburing], [AS_HELP_STRING([--with-liburing=PATH],
```

Comment:
Need to add

```
BuildRequires: liburing-devel
```

to the geopm-service.spec.in file.  Looks like this is teh right name for both SLES and RHEL rpm naming schemes.

Reply:
Also add the `--enable-io-uring` config option and `Requires: liburing` to that specfile.